vms/platformvm/block/proposal_block.go

```diff
@@ -39,6 +52,10 @@ func (b *BanffProposalBlock) Timestamp() time.Time {
 	return time.Unix(int64(b.Time), 0)
 }
 
+func (b *BanffProposalBlock) Txs() []*txs.Tx {
+	return append(b.Transactions, b.Tx)
```

Comment:
Do we care that this method is read + write? It seems odd to modify `b.Transactions` here

Reply:
I don't think `b.Transactions` gets modified here

Reply:
I think it's safer to not modify- adjusted